lib/src/native_updater.dart

```diff
@@ -94,13 +105,12 @@ class NativeUpdater {
   }
 
   void _showMaterialAlertDialog() async {
-    developer.log('Playstore URL: ${_playStoreUrl ?? ''}');
-
     /// In App Update Related
     try {
       AppUpdateInfo _updateInfo = await InAppUpdate.checkForUpdate();
 
-      if (_updateInfo.updateAvailability == UpdateAvailability.updateAvailable) {
+      if (_updateInfo.updateAvailability ==
```

Comment:
that's not a good formatting, does not seems to follow the standard. A single line is preferred in this case.